client/js/components/map/map/DpOlMapEditFeature.vue

```diff
@@ -210,7 +210,11 @@ export default {
     getZIndex (element) {
       const z = window.getComputedStyle(element).getPropertyValue('z-index')
       if (isNaN(z)) {
-        return this.getZIndex(element.parentNode)
+        if (element.nodeName !== 'HTML') {
+          return this.getZIndex(element.parentNode)
+        } else {
+          return 1
+        }
```

Comment:
Certainly not a blocker but i find ternaries to be more readable by now. Also, i'd suggest to move the critical condition at the start of the expression.

```suggestion
        return (element.nodeName === 'HTML') ? 1 : this.getZIndex(element.parentNode)
```